### PR TITLE
Update user guide and screenshots

### PR DESCRIPTION
**Pull request by tewshi**
## Checklist

- [x] The PR modified the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes in this release.
- [x] Also updates typo errors.

### CODE REVIEW

File: docs/usage_guide.rst [hunk 1]
```diff
@@ -283,6 +283,25 @@ Here you can customize the precedence of querying that price oracles will have.
...
+
+Include NFTs in graphs and total amounts
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Configure whether to include NFTs in the calculation for the total net worth and the graphs displayed in the dashboard and statistic section.
+
+NFT images rendering setting
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For privacy concerns, it is possible to allow all or only a certain list of domains for images rendered.
```
Comment (LefterisJP) [inline docs/usage_guide.rst:302]: Can you also add a link to the article describing the problem this setting aims to tackle? Here: https://medium.com/@alxlpsc/critical-privacy-vulnerability-getting-exposed-by-metamask-693c63c2ce94

File: docs/usage_guide.rst [hunk 2]
```diff
@@ -52,14 +52,14 @@ All accounts are created in the rotki directory, see the section :ref:`rotki_dat
...
 To do that you must follow the instructions described in the previous section, add the **API key/secret** and :red:`use the same password` that you were using for the local user of that database. If the password is not the same, opening the database will fail. Do not use the password of your `rotki.com <https://rotki.com/>`__ account here.
 
 .. image:: images/rotki_premium_signup_failed.png
    :alt: Sign-up with existing premium subscription using a wrong password
    :align: center
 
-See the section :ref:`sync-data-with-rotki-server` to know more about how the premium subscription will behave with multiple accounts/devices and how to sync your data with Roki Server (this option is disabled by default).
+See the section :ref:`sync-data-with-rotki-server` to know more about how the premium subscription will behave with multiple accounts/devices and how to sync your data with Rotki Server (this option is disabled by default).
```
Comment (LefterisJP) [inline docs/usage_guide.rst:62]: We need to keep rotki lowercase

```suggestion
See the section :ref:`sync-data-with-rotki-server` to know more about how the premium subscription will behave with multiple accounts/devices and how to sync your data with the rotki Server (this option is disabled by default).
```

File: docs/usage_guide.rst [hunk 3]
```diff
@@ -997,6 +1010,32 @@ An estimation of the value of the NFTs you own is counted into your total net wo
 
 If a price cannot be found for an NFT asset or if you want to change the calculated price estimate you can easily set the price for an NFT asset manually. You can do this by either clicking on the ``>`` in the NFTs table in the dashboard or by going to ``Blockchains & Accounts`` -> ``Non-fungible balances``. And then click on the pen icon for the NFT you are interested in.
 
+For privacy concerns, it is possible to allow all or only a certain list of domains for images rendered, this can be done here by clicking on this icon
```
Comment (LefterisJP) [inline docs/usage_guide.rst:1013]: add the aforementioned link here too

File: docs/websockets.rst [hunk 4]
```diff
@@ -156,7 +156,7 @@ EVM Token Detection
 
 While we are processing EVM transactions new tokens may be detected and added to the database. Some of them can be spam tokens. Using this message we can let the frontend know which tokens are detected. Then they can in turn allow the user to see an aggregated list of all detected tokens and using that list, easily mark spam assets if any.
 
-This also contains two optional, mutually excluse keys. If one exists the other shold not. But also both can be missing.
+This also contains two optional, mutually exclude keys. If one exists the other should not. But also both can be missing.
```
Comment (LefterisJP) [inline docs/websockets.rst:159]:
```suggestion
This also contains two optional, mutually excluded keys. If one exists the other should not. But also both can be missing.
```

**Review by LefterisJP — COMMENTED** [review]
Some small comments

File: docs/usage_guide.rst (hunk 1 above)
Comment (tewshi) [reply docs/usage_guide.rst:302]: Ok

File: docs/usage_guide.rst [hunk 5]
```diff
@@ -755,17 +770,17 @@ This is solely for importing generic trades. The expected file format is **CSV**
...
 A sample generic trades template can be found below
 
-    .. csv-table:: Rotki Generic Trades Template
+    .. csv-table:: rotki Generic Trades Template
        :file: files/rotki_generic_trades.csv
        :class: longtable
        :header-rows: 1
 
-Rotki Generic Events Import
+rotki Generic Events Import
```
Comment (LefterisJP) [inline docs/usage_guide.rst:778]: In titles and in start of a sentence it looks weird to be lowercase as I told you on discord

```suggestion
Rotki Generic Events Import
```

Comment (tewshi) [reply docs/usage_guide.rst:778]: alright then, I'll revert these ones

**Review by LefterisJP — APPROVED** [review]
LGTM.

To be honest I am not sure about capitalizing rotki anywhere, even in start of a sentence but let's go with it for now as it's an easy rule.

**Pull request merged**